packages/node_modules/pouchdb-replicator/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "pouchdb-replicator",
+  "version": "2.3.4",
+  "description": "A PouchDB plug-in that simulates CouchDB's replicator database daemon.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/pouchdb/pouchdb-server.git"
+  },
+  "main": "lib/index.js",
+  "keywords": [
+    "pouch",
+    "pouchdb",
+    "couch",
+    "couchdb",
+    "replication",
+    "replicator",
+    "replicate"
+  ],
+  "license": "Apache-2.0",
+  "author": "Marten de Vries",
+  "dependencies": {
+    "equals": "^1.0.5",
+    "extend": "^3.0.0",
+    "pouchdb-plugin-error": "^1.0.1",
+    "pouchdb-promise": "^6.1.2",
+    "pouchdb-system-db": "^1.0.4",
+    "pouchdb-validation": "^1.2.1",
+    "promise-nodify": "^1.0.2",
+    "random-uuid-v4": "0.0.6"
+  },
+  "devDependencies": {
+    "pouchdb-plugin-helper": "^3.0.0"
+  },
+  "optionalDependencies": {}
```

Comment:
Hmm, probably forgot that while moving to Github. Anyway, I always meant for this to be under Apache-2.0; or even ISC license if anyone prefers.